src/libs/actions/Report.js

```diff
@@ -1624,12 +1637,29 @@ function toggleEmojiReaction(reportID, reportAction, emoji, paramSkinTone = pref
 
 /**
  * @param {String|null} url
+ * @param {Boolean} isAuthenticated
+ * @param {Boolean} isOffline
  */
-function openReportFromDeepLink(url) {
+function openReportFromDeepLink(url, isAuthenticated, isOffline) {
+    const route = ReportUtils.getRouteFromLink(url);
+    const reportID = ReportUtils.getReportIDFromLink(url);
+
+    if (reportID && !isAuthenticated) {
+        // Check if it's a public room to open it as an anonymous user
```

Comment:
This comment is confusing since we're not checking got public room nor an anonymous user

Reply:
Updating it to make it clearer